src/trio/_tests/test_testing_raisesgroup.py

```diff
@@ -118,8 +118,7 @@ def test_catch_unwrapped_exceptions() -> None:
     with pytest.raises(
         ValueError, match="^You cannot specify multiple exceptions with"
     ):
-        with RaisesGroup(SyntaxError, ValueError, allow_unwrapped=True):
-            ...
+        RaisesGroup(SyntaxError, ValueError, allow_unwrapped=True)
```

Comment:
For this one and other, why are we not using with block anymore?

Reply:
error is raised on construction, not on  __enter__. With a with block I needed # pragma: no cover. See review comments by a5rocks